modules/host/negotiateformcontract.go

```diff
@@ -43,6 +46,10 @@ var (
 	// which is longer than the host's maximum duration.
 	errDurationTooLong = errors.New("file contract has a duration which exceeds the duration permitted by the host")
 
+	// errEmptyFileContractTransactionSet is returned if the renter provides a
+	// nil file contract transaction set during file contract negotiation.
+	errEmptyFileContractTransactionSet = errors.New("file contract transaction set is nil - invalid!")
```

Comment:
`file contract transaction set is nil` is fine. Errors aren't supposed to have punctuation.